packages/manager/src/features/CloudPulse/Widget/CloudPulseWidgetRenderer.tsx

```diff
@@ -129,7 +127,7 @@ export const RenderWidgets = React.memo(
       !Boolean(resourceList?.length)
     ) {
       return renderPlaceHolder(
-        'Select Dashboard, Region and Resource to visualize metrics'
+        'Select dashboard and filters to visualize metrics.'
```

Comment:
Same comment as above -- could this be more appropriate? I am not sure.
```suggestion
        'Select a dashboard and filters to visualize metrics.'
```